### DISCUSSION TITLE
Setting Up Multiple Tenants on a Single Instance

### DISCUSSION THREAD

**Discussion by prashanthShiksha** [Sunbird-Obsrv/Community#164] [Query] [answered]
We want to set up two different tenants for two organizations on a single instance. For example, for Shikshagraha, the tenant is "shiksha" and for Shikshalokam, the tenant is "demotenant1".

To access the state-related slug on the Admin Dashboard, we have to set the `sunbird_super_admin_slug` value to the organization tenant value. We tried to set the values in the following ways:

1. sunbird_super_admin_slug: "shiksha"
   sunbird_super_admin_slug: "demotenant1"

2. sunbird_super_admin_slug: ["shiksha", "demotenant1"]

However, we are getting an error.Could you please help us resolve this issue?
cc @princegupta1131, @Abhign , @rajeevsathis , @sowmya-dixit

**Comment by prashanthShiksha** [comment]
@rajeevsathish

**Reply by Abhign** [reply]
@princegupta1131 Can you check this discussion thread and respond to it once?

CC: @vinukumar-vs

**Reply by princegupta1131** [reply]
Hi @prashanthShiksha,
It will take a single channel as input. For multiple values, changes will be required in the Portal.
CC: @vinukumar-vs

**Reply by prashanthShiksha** [reply]
Hi @princegupta1131 , 
could you please show us where exactly the changes have to be made?
cc @siddhishinde0723 , @vivek-m-gowda

**Reply by SanthoshVasabhaktula** [reply, marked as answer]
@prashanthShiksha @princegupta1131 - Please move this thread to sunbird-ed discussion forum
